graphql_api/types.py

```diff
@@ -682,3 +682,13 @@ class ZeroShotNRecords(graphene.ObjectType):
 class ZeroShotNRecordsWrapper(graphene.ObjectType):
     duration = graphene.Float()
     records = graphene.List(ZeroShotNRecords)
+
+
+class ModelProviderInfoResult(graphene.ObjectType):
+    name = graphene.String()
+    revision = graphene.String()
+    link = graphene.String()
+    date = graphene.Float()
+    size = graphene.Int()
```

Comment:
i think this is the byte size of the model. For other size objects, we decided to use Float to ensure the value fits.(see byte_size field)

Reply:
changed